### PR TITLE
Update blog dependencies

### PR DESCRIPTION
**Pull request by Vik-Th0r**

### CODE REVIEW

File: _algolia.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 algolia:
     application_id: '5IGTHBX5JS'
     index_name:     'blog_eleven'
-    excluded_files:
+    extensions_to_index:
```
Comment (VEBERArnaud) [inline _algolia.yml:5]:
```suggestion
    files_to_exclude:
```

based on migration guide https://community.algolia.com/jekyll-algolia/migration-guide.html#options

> excluded_files has been renamed to files_to_exclude

File: _config.yml [hunk 2]
```diff
@@ -91,10 +91,10 @@ plugins:
     - jekyll-paginate
     - jekyll-multiple-languages-plugin
     - jekyll-seo-tag
-    - algoliasearch-jekyll
     - jekyll-feed
     - amp-jekyll
     - uglifier
+    - jekyll-algolia
```
Comment (VEBERArnaud) [inline _config.yml:97]:
```suggestion
```

from migration guide https://community.algolia.com/jekyll-algolia/migration-guide.html#renaming

> Note that you no longer need to add the plugin to the list of plugins in your _config.yml, this is now taken care of at the Gemfile level. The other important change is that the new version requires at least Ruby 2.3 and Jekyll 3.6 to work.

**Review by VEBERArnaud — CHANGES_REQUESTED** [review]
some changes requested, but the overall update process looks goof 😉

File: Gemfile.lock [hunk 3]
```diff
@@ -1,123 +1,134 @@
 GIT
-  remote: https://github.com/eko/jekyll-multiple-languages-plugin.git
-  revision: ac72749206194a11e2614c9d8e26c77d5fcf560e
+  remote: https://github.com/Vik-Th0r/amp-jekyll
```
Comment (guillemcanal) [inline Gemfile.lock:2]: Pas la bonne remote il me semble

File: Gemfile.lock [hunk 4]
```diff
@@ -1,123 +1,134 @@
...
   specs:
-    jekyll-multiple-languages-plugin (1.5.1)
-      jekyll (>= 2.0, < 4.0)
+    amp-jekyll (1.0.3)
+      fastimage (>= 1.8.0)
+      jekyll (>= 3.0, < 5.0)
+      nokogiri (>= 1.6.0)
+
+GIT
+  remote: https://github.com/Vik-Th0r/jekyll-multiple-languages-plugin
```
Comment (snroki) [inline Gemfile.lock:11]: pareil le remote devrait être le repo eleven non ?

**Review by VEBERArnaud — APPROVED** [review]
LGTM

**Pull request merged**